samples/isilon_v250_ops_410.yaml

```diff
@@ -117,18 +123,24 @@ spec:
       # Leave as blank to consider all nodes
       # Allowed values: map of key-value pairs
       # Default value: None
-      # Examples:
-      #   node-role.kubernetes.io/master: ""
       nodeSelector:
+      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
       #  node-role.kubernetes.io/master: ""
+      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/control-plane taint
+      #  node-role.kubernetes.io/control-plane: ""
 
       # tolerations: Define tolerations for the controller deployment, if required.
       # Default value: None
+      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
```

Comment:
This comment should come above line no 136

Reply:
moved